src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNIProxy.cs

```diff
@@ -381,6 +380,7 @@ internal class DataSource
         private const string Slash = @"/";
         private const string PipeToken = "pipe";
         private const string LocalDbHost = "(localdb)";
+        private const string LocalDbHost_NamedInstance = @"np:\\.\pipe\LOCALDB#";
```

Comment:
NIT: please, update it in other files too.
```suggestion
        private const string LocalDbHost_NP = @"np:\\.\pipe\LOCALDB#";
```